spec/requests/affiliates_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe "GET /affiliates" do
     it "works! (now write some real specs)" do
       get affiliates_path
-      expect(response).to have_http_status(200)
+      expect(response).to have_http_status(302)
```

Comment:
This means it's not logging in properly. Add an auth sign in call before this call. :)